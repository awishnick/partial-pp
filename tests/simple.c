// RUN: -DTESTSYM
#ifdef TESTSYM
// this is defined
#endif

#ifdef NOT_TESTSYM
// this is not defined
#endif
