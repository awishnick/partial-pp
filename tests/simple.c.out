// RUN: -DTESTSYM
// this is defined

#ifdef NOT_TESTSYM
// this is not defined
#endif
